File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Calendars/handleoccurrencesforexceptions.rb
module Asposetasksjava
  module HandleOccurrencesForExceptions
    def initialize()
        except = Rjb::import('com.aspose.tasks.CalendarException').new
        except.setEnteredByOccurrences(true)
        except.setOccurrences(5)
        except.setType(Rjb::import('com.aspose.tasks.CalendarExceptionType').YearlyByDay)

        puts "Set Occurences for exeptions"
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithCalendarExceptions/HandleOccurrencesForExceptions.php
<?php
namespace Aspose\Tasks\WorkingWithCalendarExceptions;

use com\aspose\tasks\CalendarException as CalendarException;
use com\aspose\tasks\CalendarExceptionType as CalendarExceptionType;

class HandleOccurrencesForExceptions{

    public static function run($dataDir=null){

        $except = new CalendarException();
        $except->setEnteredByOccurrences(true);
        $except->setOccurrences(5);

        $calendarExceptionType=new CalendarExceptionType();
        $except->setType($calendarExceptionType->YearlyByDay);

        print "Set Occurences for exeptions".PHP_EOL;
    }
}
?>